core/deployment/src/main/java/io/quarkus/deployment/builditem/GeneratedFileSystemResourceHandledBuildItem.java

```diff
@@ -0,0 +1,10 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+/**
+ * Marker used only to ensure that the file system resources where properly written in dev-mode
+ */
+public final class GeneratedFileSystemResourceHandledBuildItem extends SimpleBuildItem {
```

Comment:
Why not just ask for GeneratedFileSystemResourceBuildItem when augmenting dev mode?